CONTRIBUTING.md

```diff
@@ -50,6 +50,7 @@ If this doesn't work (rare), then:
 If you'd like to use your modified ReScript like an end-user, try:
 
 ```sh
+npm uninstall -g bs-platform
 BS_TRAVIS_CI=1 npm install -g .
```

Comment:
why uinstalled is needed?

Reply:
I don't know. You can try it yourself. I updated a certain file, ran ninja config and build, and did the installation. A second time doing so doesn't work without first removing bs-platform. Didn't use to be this way. npm 7.0.3

Reply:
I am not on npm 7, can you add a note that for npm 7, you may need uninstall it first